### ISSUE TITLE
Uncaught TypeError: Cannot perform DataView constructor on a detached ArrayBuffer

### ISSUE THREAD

**Issue by sunsetkookaburra** [sunsetkookaburra/bitbin#1]
![image](https://user-images.githubusercontent.com/55334808/201519016-eb49601a-2d1c-4a58-970c-76bb3bf33219.png)

**Comment by sunsetkookaburra** [comment]
reproduced here
```ts
import { Uint8 } from "./formats/mod.ts";
const f = Deno.openSync("bug.ts");
try {
  while (true) await Uint8.readFrom(f);
} catch (e) {
  console.log(e);
}
await Uint8.readFrom(f);
f.close();

```

**Comment by sunsetkookaburra** [comment]
fixed in 0b07ddc5acaf230a391dfb3e8e3c157df3f9367b
